src/_cffi_src/openssl/ssl.py

```diff
@@ -590,6 +599,21 @@
 static const long TLS_ST_OK = 0;
 #endif
 
+#if defined(OPENSSL_NO_DTLS1_2_METHOD) || \
```

Comment:
Yes there really are this many ways to disable DTLS 1.2 support. Probably nothing uses them, but 🤷‍♂️